vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -1,26 +1,51 @@
 packages:
   ubuntu:
-    packages:
-     - curl
-     - dnsmasq
-     - nmap
-     - ovmf
-     - patch
-     - psmisc
-     - wget
-     - libvirt-bin
-     - libvirt-clients
-     - libvirt-dev
-     - jq
-     - unzip
-     - yarn
-     - genisoimage
-     - qemu-kvm
-     - libguestfs-tools
-     - gir1.2-polkit-1.0
-     - libpolkit-agent-1-0
-     - libpolkit-backend-1-0
-     - libpolkit-gobject-1-0
+    bionic:
+      packages:
+        - curl
+        - dnsmasq
+        - nmap
+        - ovmf
+        - patch
+        - psmisc
+        - wget
+        - libvirt-clients
+        - libvirt-dev
+        - libvirt-bin
+        - bridge-utils
+        - jq
+        - unzip
+        - yarn
+        - genisoimage
+        - qemu-kvm
+        - libguestfs-tools
+        - gir1.2-polkit-1.0
+        - libpolkit-agent-1-0
+        - libpolkit-backend-1-0
+        - libpolkit-gobject-1-0
+    focal:
+      packages:
+        - curl
+        - dnsmasq
+        - nmap
+        - ovmf
+        - patch
+        - psmisc
+        - wget
+        - libvirt-clients
+        - libvirt-dev
+        - libvirt-daemon
+        - libvirt-daemon-system
+        - bridge-utils
+        - jq
+        - unzip
+        - yarn
+        - genisoimage
+        - qemu-kvm
+        - libguestfs-tools
+        - gir1.2-polkit-1.0
+        - libpolkit-agent-1-0
+        - libpolkit-gobject-1-0
```

Comment:
If the difference between these two blocks is a single package (`libpolkit-backend-1-0`), can't we default packages that are the same for both 18.04 & 20.04?

Reply:
done, thanks